src/utils/formatters.js

```diff
@@ -0,0 +1,4 @@
+export function dateformatter(date) {
+    const d = new Date(date);
+    return `${d.getDate()}-${d.getMonth()}-${d.getFullYear()}`;
+};
```

Comment:
Nice util function! Any small functional extractions like this, consider building with TDD.